test/sanity/ignore.txt

```diff
@@ -983,6 +983,8 @@ lib/ansible/modules/cloud/openstack/os_project_access.py validate-modules:doc-mi
 lib/ansible/modules/cloud/openstack/os_project_access.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/cloud/openstack/os_project_info.py validate-modules:doc-missing-type
 lib/ansible/modules/cloud/openstack/os_project_info.py validate-modules:parameter-type-not-in-doc
+lib/ansible/modules/cloud/openstack/os_qos_policy_rule.py validate-modules:doc-missing-type
+lib/ansible/modules/cloud/openstack/os_qos_policy_rule.py validate-modules:undocumented-parameter
```

Comment:
why are these validations being ignored?

Reply:
Because the shared fragment this module extends and the shared lib used by all openstack modules has been hiding this parameter. I could fix it but that would require changing it in all openstack modules, which is probably not what authors had in mind.

Reply:
Instead of ignoring these, open a new PR to fix the issue(s) in the shared fragment(s). Once that is merged this PR should pass without needing new ignore entries.